src/views/Recorder/EmptyState.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useRef } from 'react'
+import { css } from '@emotion/react'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { DiscIcon } from '@radix-ui/react-icons'
+import { Button, Flex, Heading, Text, TextField } from '@radix-ui/themes'
+import { useForm } from 'react-hook-form'
+import { z } from 'zod'
+
+import { FieldGroup } from '@/components/Form'
+
+interface EmptyStateProps {
+  isLoading: boolean
+  onStart: (url?: string) => void
+}
+
+const RecorderEmptyStateSchema = z.object({
+  url: z.string(),
+})
+
+type RecorderEmptyStateFields = z.infer<typeof RecorderEmptyStateSchema>
+
+export function EmptyState({ isLoading, onStart }: EmptyStateProps) {
+  const inputRef = useRef<HTMLInputElement | null>(null)
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<RecorderEmptyStateFields>({
+    resolver: zodResolver(RecorderEmptyStateSchema),
+    defaultValues: {
+      url: '',
+    },
+    shouldFocusError: false,
+  })
+
+  const { ref, ...inputProps } = register('url')
+
+  const onSubmit = ({ url }: RecorderEmptyStateFields) => {
+    onStart(url)
+  }
+
+  useEffect(() => {
+    window.requestAnimationFrame(() => {
+      inputRef.current?.focus()
+    })
+  }, [])
+
+  return (
+    <Flex direction="column" align="center" gap="2" pt="90px">
+      <Heading
+        size="8"
+        css={css`
+          font-weight: 400;
+        `}
+      >
+        Record your user flow
+      </Heading>
+      <Text color="gray" size="1">
+        Once you begin recording, requests will appear in this area
+      </Text>
+      <form
+        onSubmit={handleSubmit(onSubmit)}
+        css={css`
+          margin-top: var(--space-6);
+        `}
+      >
+        <FieldGroup
+          name="url"
+          label="Starting URL"
+          hint="Enter the URL of the website or service you want to test"
```

Comment:
Should we maybe mention `Optionally enter....` ? 🤔

Reply:
@Llandy3d I think it's better to start with the action, so I'd maybe suggest appending `(optional)` to the hint or the label, but I'm not sure it's really needed. Let's wait for the final design and discuss it again

Reply:
Makes sense!
It was not clear on first impact that the field could be empty but looking forward to the final design 👀